cfme/infrastructure/virtual_machines.py

```diff
@@ -1071,113 +1174,118 @@ class VmAllWithTemplatesDetails(CFMENavigateStep):
     VIEW = InfraVmDetailsView
```

Comment:
Toolbar is different between template and VM view, you'll need a different view here